templates/partials/content-post.php

```diff
@@ -4,6 +4,7 @@
     <header class="page-header">
       <h1><?php the_title() ?></h1>
       <div class="entry-meta date">
+        <?php the_author() ?>,
```

Comment:
I think we should make it a link to the author page, eg http://digital.nhs.uk/author/joe-mcgrath